packages/containers/src/ComponentForm/ComponentForm.component.js

```diff
@@ -165,6 +165,10 @@ export class TCompForm extends React.Component {
 			customRegistry: props.customTriggers,
 			headers: config.headers,
 			lang: props.lang,
+			security: {
+				CSRFTokenCookieKey: this.props.CSRFTokenCookieKey,
```

Comment:
Is there a reason to use `this.props` instead of `props` only on these 2? They should be the same, but it looks like they are not meant to be.